### PR TITLE
Add alert to Deckhouse CLI page

### PR DESCRIPTION
**Pull request by elenashliaga**
## Description

Add alert about Community Edition.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: chore
summary: Add info about using `d8 d` commands.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @elenashliaga. I've started the workflow for label 'deploy/web/test'...

File: docs/documentation/pages/DECKHOUSE-CLI_RU.md [hunk 1]
```diff
@@ -11,6 +11,11 @@ Deckhouse CLI — это интерфейс командной строки дл
     Например, в кластере можно выполнить `kubectl get pods` как `d8 k get pods`.
 * `d8 d` — команды, отвечающие за доставку по аналогии с утилитой `werf`.  
     Например, вместо `werf plan --repo registry.deckhouse.io` можно выполнить `d8 d plan --repo registry.deckhouse.io`.
+
+    {% alert level="warning" %}
+    Группа команд `d8 d` не доступна для редакции Community Edition (CE).
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-CLI_RU.md:16]:
```suggestion
    Группа команд `d8 d` не доступна для Community Edition (CE).
```

**Review by MariaMakeeva — APPROVED** [review]

File: docs/documentation/pages/DECKHOUSE-CLI.md [hunk 2]
```diff
@@ -10,6 +10,11 @@ On the command line, the utility can be invoked using the `d8` alias. All the co
     For example, `d8 k get pods` is the same as `kubectl get pods`.
 * `d8 d` — the range of delivery-related commands (see the `werf` tool).  
     For example, you can run `d8 d plan --repo registry.deckhouse.io` instead of `werf plan --repo registry.deckhouse.io`.
+
+    {% alert level="warning" %}
+    The `d8 d` command group is not available for Community Edition (CE).
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-CLI.md:15]:
```suggestion
    The `d8 d` command group is not available for Community Edition (CE) and Basic Edition (BE).
```

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [8131](https://github.com/deckhouse/deckhouse/pull/8131) to the branch [release-1.59](https://github.com/deckhouse/deckhouse/tree/release-1.59) successful!

**Pull request merged**